### PR TITLE
[#11830] User documentation: don't use bold to refer to UI text

### LINKED ISSUE #11830: User documentation: don't use bold to refer to UI text
Current: the user documentation uses **bold text** to refer to text in the UI. e.g.,
image

Suggestion: Use a different style to differentiate UI text from the rest e.g., `this style`

Reason: Bold is normally used to make the text stand out by adding more visual weight e.g., to indicate more important parts of the text. By using it for the above purpose, we lose the opportunity to emphasize important parts of the user documentation using bold formatting.

What do others think?

### PR DESCRIPTION
**Pull request by u7305725**
Fixes #11830

Add css style for the bold text in instructor-help pages.

The [previous pull request](https://github.com/TEAMMATES/teammates/pull/11954#issue-1363905741) is simply switch all "b" tags to "marker" tags in related html which may give users wrong idea of these words. While in this pull request, i used CSS to change the style of "b" tags. The CSS now make the texts in "b" tags differentiate but not stand out more than the regular text. 

These are the parameter for b tags:
    1. color: #908B8B;
    2. background-color: #E6E9E9;
    3. font-size: 13px;
    4. padding-left: 1.0%;
    5. padding-right: 1.0%;

### CODE REVIEW

**Review by zhaojj2209 — CHANGES_REQUESTED** [review]
This solution fails to address the core issue. The entire reason why issue #11830 is created is because we would like to move away from using bold formatting UI text, so that we can use bold formatting to emphasise other parts of the text instead. Changing the style of the `<b>` tag does not change the fact that we are still using the same tag; with this change, any attempt at bolding text for emphasis in these pages will result in the text being formatted as if it were UI text.

There has already been another solution proposed (and being worked on in #11954) which involves creating a custom HTML tag to be used specifically for UI text. Please look into that approach if you would like to continue tackling this issue.
In fact, this PR was created without checking whether the issue is being worked on by someone else, which goes against etiquette. Please check with the creator of #11954 for permission to continue working on the issue.

**Comment by u7305725** [comment]
> This solution fails to address the core issue. The entire reason why issue #11830 is created is because we would like to move away from using bold formatting UI text, so that we can use bold formatting to emphasise other parts of the text instead. Changing the style of the `<b>` tag does not change the fact that we are still using the same tag; with this change, any attempt at bolding text for emphasis in these pages will result in the text being formatted as if it were UI text.
> 
> There has already been another solution proposed (and being worked on in #11954) which involves creating a custom HTML tag to be used specifically for UI text. Please look into that approach if you would like to continue tackling this issue. In fact, this PR was created without checking whether the issue is being worked on by someone else, which goes against etiquette. Please check with the creator of #11954 for permission to continue working on the issue.

Thanks for your classification. I will continue work on this issue. Meanwhile i will get contact with previous person who were doing this.

**Comment by u7305725** [comment]
Hi everyone, i switch all the b tags into a span with class "my_tag"  so that i can set the style for .my_tag instead of set style for bold. 

From now on, the bold tags will be free to be used. Also, developers can use <span class="my_tag">some text</span> to call the my custom style.

**Review by zhaojj2209 — CHANGES_REQUESTED** [review]
Thanks for making the custom style. However, the name `my-tag` tells me nothing about what this custom style should be used for. Please change the name to something more descriptive (e.g. `ui-text`).

The replacement of bold formatting with this new custom style is also more than just a simple find-and-replace. As has already been mentioned in the issue, custom style should only be used for text in the UI; all other instances of bold formatting should not be replaced. Please take the effort to go through the documentation pages manually and revert the changes for any text that's clearly supposed to stay bold, such as section titles. Some examples:
- All numbered headings in the Getting Started guide
photo_2022-10-20_15-21-47
image


- Headings of bullet points that are not UI-related
photo_2022-10-20_15-40-03
image
image

**Comment by u7305725** [comment]
@zhaojj2209 Thankyou for pointing out. I will do this immediately and provide the better one within 2 days

**Comment by u7305725** [comment]
Rename custom tag my_tag to ui-text . Correct the use of ui-text. (Previously headings are all in ui-text tag, now they are return b tag)

**Comment by zhaojj2209** [comment]
Lint tests are failing, please address the failing tests.

**Comment by u7305725** [comment]
I have addressed the failing test,please review it tks

File: src/web/app/pages-help/instructor-help-page/instructor-help-getting-started/instructor-help-getting-started.component.html [hunk 1]
```diff
@@ -26,22 +26,22 @@ <h2 id="course-setup">1. Set up a course</h2>
...
         </tm-example-box>
       </li>
       <li>
         <b><a [tmRouterLink]="instructorHelpPath" [queryParams]="{questionId: CoursesSectionQuestions.COURSE_ADD_STUDENTS, section: Sections.courses}">Enroll students in the course</a></b><br>
-        Go to the <b>Courses</b> page and click the <button class="btn btn-light btn-sm">Enroll</button> button of the corresponding course.<br>
+        Go to the <span class="ui-text">Courses</span> page and click the <button class="btn btn-light btn-sm">Enroll</button> button of the corresponding course.<br>
         Students can be enrolled into teams (e.g., project groups) and sections (e.g., tutorial classes, lecture groups) to facilitate giving feedback in and among these smaller groups.<br>
-        TEAMMATES will <b>not</b> automatically notify students that they have been enrolled, as there is nothing for students to do in TEAMMATES until a session opens for submissions.<br>
+        TEAMMATES will <span class="ui-text">not</span> automatically notify students that they have been enrolled, as there is nothing for students to do in TEAMMATES until a session opens for submissions.<br>
```
Comment (zhaojj2209) [inline src/web/app/pages-help/instructor-help-page/instructor-help-getting-started/instructor-help-getting-started.component.html:39]:
```suggestion
        TEAMMATES will <b>not</b> automatically notify students that they have been enrolled, as there is nothing for students to do in TEAMMATES until a session opens for submissions.<br>
```

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html [hunk 2]
```diff
@@ -478,7 +478,7 @@ <h4 *ngIf="displaySubsection(0, 10)">Setting Up Questions</h4>
         <i>team's perception of how much the student contributed</i>.
       </li>
       <li>
-        <b>Diff</b>: The difference between the claimed contribution (CC) and the perceived contribution (PC). This value can be used to identify those who have over/under-estimated their own contribution.
+        <span class="ui-text">Diff</span>: The difference between the claimed contribution (CC) and the perceived contribution (PC). This value can be used to identify those who have over/under-estimated their own contribution.
```
Comment (zhaojj2209) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html:481]:
```suggestion
        <b>Diff</b>: The difference between the claimed contribution (CC) and the perceived contribution (PC). This value can be used to identify those who have over/under-estimated their own contribution.
```

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<div>
+<div xmlns="http://www.w3.org/1999/html">
```
Comment (zhaojj2209) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html:1]: Any reason why the XML namespace is added here? Do revert this if it's not necessary.

**Review by zhaojj2209 — COMMENTED** [review]
Almost there, just a few more changes to revert

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html (hunk 3 above)
Comment (u7305725) [reply src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html]: This is not useful, i have reverted it

**Comment by u7305725** [comment]
Thanks @zhaojj2209 ,i have commited your changes

**Review by zhaojj2209 — APPROVED** [review]
LGTM, thanks for the changes

**Review by ziqing26 — APPROVED** [review]
LGTM as well, thank you! Kindly update the branch for it to be merged.

**Comment by u7305725** [comment]
@hhdqirui @jianhandev  are the changes look fine? Please approve if you think they are fine

**Pull request merged**